data/card_catalogue/boss/shared-boss.json

```diff
@@ -1,42 +1,41 @@
 [
-  {
-    "title": "Enough Playing Around",
-    "upgrade": "none",
-    "tier": "gold",
-    "details": {
-      "rules": "Cast when unconscious. Return to full health",
-      "flavour": "Not bad. But watch this...",
-      "image": "assets/core/card/illustration/Platzfalt Step.png"
-    }
-  },
   {
     "title": "I've Had Enough",
     "upgrade": "none",
     "tier": "gold",
     "details": {
-      "rules": "Cast this card only during your turn. At the start of your next turn, teleport to        safety and leave behind a piece of valuable loot.",
+      "rules": {
+        "active": "Cast this card only during your turn. At the start of your next turn, teleport to safety and leave behind a piece of valuable loot.",
```

Comment:
unsure if we're keeping the generic boss cards in but auto-formatted them anyway

Reply:
I suppose we'll have to see once we start designing the boss fights. I think these ones all serve a useful level design purpose

Reply:
`I've Had Enough` -> We want you to fight this guy but we don't want him to die this session
`Attrition` -> We want to make sure that this boss fight isn't just them hitting you every turn until they die
`Take it Outside` -> There are nearby plot relevant characters which make the decision to fight unrealistic

Reply:
Sounds good